<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>donorsTable</class>
 <widget class="QMainWindow" name="donorsTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(142, 157, 204);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="donRepTable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>460</width>
      <height>230</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: #D9DBF1;
    color: #000000;
    border-radius: 20px;
    font-family: 'Gogh';
    font-weight: 500;
    font-size: 15px;
}

QTableWidget::item:hover {
    border-right: 0.5px solid #E5E5E5;
    background-color: #A9B9E3;
}

QTableWidget::item:selected {
    background-color: #A9B9E3;
    border: none;
}

QTableWidget::item {
    border-right: 1px solid #8E9DCC;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="universalButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}
QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}</string>
    </property>
    <property name="text">
     <string>Universal donors | recipients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generalButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}
QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}</string>
    </property>
    <property name="text">
     <string>General table of donors | recipients</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>340</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../images/DonorsRec.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
